File: aula3go/Ex1.go
package main

import "fmt"

func main() {
	var arr1 = [8]int{0, 1, 2, 3, 4, 5, 6, 7}
	var x = 0
	var y = 0

	fmt.Print("Digite a posição de x: ")
	fmt.Scan(&x)
	fmt.Print("digite a posição de y: ")
	fmt.Scan(&y)

	var resultado = arr1[x] + arr1[y]

	fmt.Println("a soma dos dois é igual a: ", resultado)
}

File: Aula3java/src/Ex1.java
import java.util.ArrayList;
import java.util.Scanner;

public class Ex1 {
    Scanner sc = new Scanner(System.in);
    public void Achar(){
        ArrayList<Integer> valores = new ArrayList<Integer>();

        for (int i = 0; i <= 8; i++){
            valores.add(i);
        }
        System.out.print("digite a posicao de x: ");
        int x = sc.nextInt();
        System.out.print("digite a posicao de y: ");
        int y = sc.nextInt();
        int resultado = valores.get(x) + valores.get(y);
        System.out.println("O valor na posicao x é igual a " + valores.get(x) + " O valor na posicao y é igual a " + valores.get(y) + " e a soma dos dois é igual a " + resultado );


    }
}
